packages/mobile/src/verify/VerificationLoadingScreen.tsx

```diff
@@ -117,6 +127,10 @@ class VerificationLoadingScreen extends React.Component<Props> {
         </View>
         <VerificationFailedModal
           verificationStatus={verificationStatus}
+          retryWithForno={retryWithForno}
+          fornoMode={fornoMode}
```

Comment:
Have to keep track of forno mode to avoid prompting users who are already in forno mode